docs/rules/no-multi-comp.md

```diff
@@ -69,6 +69,55 @@ class HelloJohn extends React.Component {
 module.exports = HelloJohn;
 ```
 
+### `ignorePrivate`
```

Comment:
"private" might not be the best name here. maybe "ignoreNonExported"?

Reply:
`ignoreNonExported` feels a little clunky to me but I can't think of anything better. just one of those hard problems in software.

Reply:
yeah totally :-) "ignoreInternal"?

Reply:
Sounds good to me, switched over.